### PR TITLE
alter platform support table to reflect current

### PR DESCRIPTION
**Pull request by vtjnash**
Win7 is EOL since almost 2 years ago

AArch64, PPC, and FreeBSD are all generally stable, but we don't have people able to actively support them

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Idk if I'd agree with moving FreeBSD to Tier 2.

**Comment by DilumAluthge** [comment]
On an unrelated note, the colors need to be fixed so that e.g. "Tier 1" is always green, "Tier 2" is always orange, "Tier 3" is always red, and "Tier 4" is always black.

**Comment by DilumAluthge** [comment]
For FreeBSD, I would suggest that we do #1345 instead.

File: downloads/index.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ The platforms currently supported by Julia are listed below. They are divided in
       <td rowspan="2"> FreeBSD </td>
       <td rowspan="2"> 11.0+ </td>
       <td> x86-64 (64-bit) </td>
-      <td> <font color="green">Tier 1</font> </td>
+      <td> <font color="green">Tier 2</font> </td>
```
Comment (ararslan) [inline downloads/index.md:351]:
```suggestion
      <td> <font color="green">Tier 1</font> </td>
```
This was addressed by https://github.com/JuliaLang/www.julialang.org/pull/1345.

**Review by ararslan — CHANGES_REQUESTED** [review]

**Comment by vtjnash** [comment]
updated to fix colors

File: downloads/index.md (hunk 1 above)
Comment (ararslan) [reply downloads/index.md]: This isn't resolved

Comment (vtjnash) [reply downloads/index.md]: This is still updating FreeBSD to  be tier 2, as intended. Although I think (after this PR), it was mentioned we should also add mention of these changes to the current release notes?

**Comment by vtjnash** [comment]
Bump. This has been ready to merge for awhile now.

File: downloads/index.md [hunk 2]
```diff
@@ -304,12 +304,12 @@ The platforms currently supported by Julia are listed below. They are divided in
       <td> <font color="orange">Tier 2</font> </td>
     </tr>
     <tr>
-      <td> 7+ </td>
+      <td> 8+ </td>
       <td> i686 (32-bit) / x86-64 (64-bit) </td>
-      <td> <font color="crimson">Tier 3</font> </td>
+      <td> <font color="orange">Tier 2</font> </td>
```
Comment (ararslan) [inline downloads/index.md:309]: Is this change intended? I thought we do all of our Windows stuff with Windows 10 nowadays and don't check 8.

Comment (vtjnash) [reply downloads/index.md:309]: We don't actively check it, but it is close enough to 10 that we expect it to keep working (for now).  We will deprecate it from the list whenever it is EOL from Microsoft.

**Review by ararslan — APPROVED** [review]

File: downloads/index.md (hunk 2 above)
Comment (giordano) [reply downloads/index.md:309]: Windows 8 was EOL in January 2016. Windows 8.1 reached end of mainstream support (whatever it is) in January 2018, and will reach end of extended support in January 2013
